core-api/src/main/java/com/optimizely/ab/config/ProjectConfig.java

```diff
@@ -299,7 +296,8 @@ public Experiment getExperimentForKey(@Nonnull String experimentKey,
     }
 
 
-    public @Nullable Experiment getExperimentForVariationId(String variationId) {
+    public @Nullable
+    Experiment getExperimentForVariationId(String variationId) {
```

Comment:
Same as other comments about annotation placement for return types. This is formatting is worse.